src/scores/categorical/multicategorical_impl.py

```diff
@@ -64,6 +66,12 @@ def firm(  # pylint: disable=too-many-arguments
             supplied. The default behaviour if neither are supplied is to reduce all dims.
         weights: Optionally provide an array for weighted averaging (e.g. by area, by latitude,
             by population, custom)
+        threshold_alignment: Specifies if the left or right category includes the decision
```

Comment:
I'm not sure we have left and right categories. Try: 
Specifies whether the intervals defining the categories are left or right closed. That is, whether the decision threshold is included in the upper (left closed) or lower (right closed) category. Defaults to "right".

Reconsider if you reconsider threshold_alignment.

Reply:
updated